### PR TITLE
Add GitHub workflow cypress

### PR DESCRIPTION
**Pull request by abbyhu2000**
### Description

Add CiGroup 11 to github workflow for cypress tests of discover 2.0

## Changelog

- feat: Add cypress test workflow for discover 2.0

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: changelogs/fragments/8682.yml
Comment (ashwin-pc) [inline changelogs/fragments/8682.yml:1]: Why do we have a fragment from another PR here?

**Review by ashwin-pc — APPROVED** [review]
LGTM. Although im not the expert on the CI group matrix code so would be nice for someone who's familiar with that to also look at it. @kavilla perhaps?

File: .github/workflows/cypress_workflow.yml [hunk 1]
```diff
@@ -49,7 +51,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10,11]
```
Comment (ananzh) [inline .github/workflows/cypress_workflow.yml:54]:
```suggestion
          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
```

**Review by ananzh — APPROVED** [review]

File: .github/workflows/cypress_workflow.yml [hunk 2]
```diff
@@ -49,7 +51,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
```
Comment (Hailong-am) [inline .github/workflows/cypress_workflow.yml:54]: can you link FTR repo change for ci-group 11

**Comment by abbyhu2000** [comment]
this PR is closed, changes are tracked in a new PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8703

File: .github/workflows/cypress_workflow.yml (hunk 2 above)
Comment (abbyhu2000) [reply .github/workflows/cypress_workflow.yml:54]: Hi this PR is old, i have a new PR #8703 for the changes.
